src/DeviceInterface/initFormSubmissionsApi.js

```diff
@@ -50,7 +49,7 @@ export function initFormSubmissionsApi (forms) {
             const button = /** @type HTMLElement */(event.target)?.closest(selector)
             if (!button) return
 
-            const text = removeExcessWhitespace(button?.textContent)
+            const text = getText(button)
```

Comment:
`getText` is smarter, it also the visible content of `input[type=button/image/submit]`. It also includes `removeExcessWhitespace`.